packages/appstore/src/components/wallets-consent-popup/wallets-consent-popup.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import {
+    Button,
+    Checkbox,
+    Modal,
+    DesktopWrapper,
+    MobileWrapper,
+    MobileDialog,
+    Div100vhContainer,
+} from '@deriv/components';
+import { localize } from '@deriv/translations';
+import WalletsConsentForm from './wallets-consent-form';
+
+type TWalletsConsentPopupProps = {
+    show_wallet_consent_popup: boolean;
```

Comment:
since this component should be visible as the last step of upgrade wallets wizard , please add comment here to remove this in stitching  user story

Reply:
Done 😃 👍🏼